### DISCUSSION TITLE
libfintx for swiss banks?

### DISCUSSION THREAD

**Discussion by symasdesign** [libfintx/libfintx#21] [General]
Hi 
Since EBICS is a standard, the library should also work for Swiss banks. I don't see any Swiss bank in the "tested with" list. Now I would like to ask what requirements there are to use the library with a Swiss bank? Is there a general Getting Started guide for using the library productively?
Thanks for your help.

**Comment by leo-schick** [comment]
Hi @symasdesign 

there is a sample code for using EBICS see here: https://libfintx.github.io/docs/sample-ebics.html
If you have further questions, feel free to write them here 😉
@torstenprivate did several optimizations the EBICS implementation including A006 support. He probably can help to get running. A getting started guide would be great 👍 I’m unfortunately not so deep into EBICS yet

**Reply by iamtorsten** [reply]
Hi. Yes in general it is working. There is an example project calling Buchen.

**Reply by leo-schick** [reply]
@symasdesign where you able to test it successfully with a swiss bank? We would be happy to update the list of tested banks.
